rs/execution_environment/tests/canister_logging.rs

```diff
@@ -217,34 +228,83 @@ fn test_metrics_for_fetch_canister_logs_via_query_call() {
         ]))
         .map_or(0, |stats| stats.count)
     }
-    let (env, canister_id, controller) = setup_with_controller(wat_canister().build_wasm());
+    let (env, canister_id, user_controller) = setup_with_controller(wat_canister().build_wasm());
 
     assert_eq!(fetch_canister_logs_count(&env), 0);
-    let _ = fetch_canister_logs(&env, controller, canister_id);
+    let _ = fetch_canister_logs(&env, user_controller, canister_id);
     assert_eq!(fetch_canister_logs_count(&env), 1);
 }
 
+#[test]
+fn test_fetch_canister_logs_via_inter_canister_query_call() {
```

Comment:
I think this should be name "inter_canister_update_call/replicated_inter_canister_call", right?

Reply:
I named this test exactly how I **didn't want it to be called**! Since I'm looking into adding this feature soon I thought this case must be covered explicitly now, replicated calls should not be allowed. Nevertheless I gave it the opposite name. Good catch, thank you!